crates/ark/src/lsp/completions/sources/utils.rs

```diff
@@ -101,6 +101,7 @@ pub(super) enum CallNodePositionType {
 
 pub(super) fn call_node_position_type(node: &Node, point: Point) -> CallNodePositionType {
     match node.node_type() {
+        NodeType::Arguments => return CallNodePositionType::Name,
```

Comment:
The fixup for argument completions inside a call and custom completions such as `Sys.getenv()`.